netmiko/cisco_base_connection.py

```diff
@@ -204,25 +205,51 @@ def _autodetect_fs(self, cmd="dir", pattern=r"Directory of (.*)/"):
             "system: {} {}".format(cmd, output)
         )
 
+    def _confirm_command(
+        self, cmd="", confirm=False, confirm_response="", delay_factor=1
+    ):
+
+        self.enable()
+
+        output = self.send_command_timing(command_string=cmd, delay_factor=delay_factor)
+        # If confirm prompt detected or confirm=True, progress through action, otherwise return output
+        # Prompts are expected after confirms, so back to send_command
+        if confirm or any(confirm in output for confirm in ["[confirm]", "[y]", "[n]"]):
+            if confirm_response:
+                output += self.send_command(confirm_response, delay_factor=delay_factor)
+            else:
+                # Default RETURN if no confirm_response given
+                output += self.send_command(self.RETURN, delay_factor=delay_factor)
+        return output
+
```

Comment:
I don't think we should have a separate method for this: I think handling of this should just all be inside of the save_config method using confirm and confirm_response. 

We can potentially convert confirm_response to be regex, but if we do that we need to make the behavior consistent across all of the netmiko platforms (so that is definitely more work).